.travis.yml

```diff
@@ -14,19 +14,21 @@ before_deploy:
   - git config --global user.email "builds@travis-ci.com"
   - git config --global user.name "Travis CI"
   - git remote set-url origin https://datree-ci:$GITHUB_TOKEN@github.com/datreeio/datree.git
+  - docker pull ghcr.io/caarlos0/svu:v1.6.1
+  - export SEMVER_NUMBER=$(docker run --rm -v $PWD:/tmp --workdir /tmp ghcr.io/caarlos0/svu:v1.6.1 --strip-prefix --no-metadata)
 deploy:
   - provider: script
     skip_cleanup: true
     script: bash ./internal/release/deploy.sh
     on:
       all_branches: true
-      condition: $TRAVIS_BRANCH =~ ^(staging|main)$
+      condition: $TRAVIS_BRANCH =~ ^(staging|main|DAT-3209-semantic-versioning)$
```

Comment:
I think we want to remove this one if you done with development

Reply:
updated